container-engine-lib/lib/backend_interface/objects/logs_aggregator/components.go

```diff
@@ -0,0 +1,7 @@
+package logs_aggregator
```

Comment:
any reason to create a new golang file for these as opposed to just putting them in `logs_aggregator.go`?

Reply:
Wanted to future proof it because we might want to later add other vector components such as Transforms and Sinks, but can move it to the same file